frontend/src/components/worksheets/items/TableItem/TableItem.js

```diff
@@ -49,7 +49,7 @@ class TableItem extends React.Component<{
         var bundleInfos = item.bundles_spec.bundle_infos;
         var headerItems = item.header;
         var headerHtml = headerItems.map(function(item, index) {
-            let styleDict = index == 0 ?  {paddingLeft: 42} : {};
+            let styleDict = index == 0 ?  { paddingLeft: 28 } : { paddingLeft: 0 };
```

Comment:
need to set padding left for the rest of column names to override parent paddings